packages/angular_devkit/build_webpack/src/browser/index.ts

```diff
@@ -150,6 +152,12 @@ export class BrowserBuilder implements Builder<BrowserBuilderOptions> {
           return;
         }
         const webpackCompiler = webpack(webpackConfig);
+
+        // TODO: fix webpack typings.
+        // tslint:disable-next-line:no-any
+        (webpackCompiler as any).inputFileSystem = new WebpackFileSystemHostAdapter(
```

Comment:
The watch file system will be unaffected and I don't think caching will be used.  This webpack plugin is always applied before anything else: https://github.com/webpack/webpack/blob/master/lib/node/NodeEnvironmentPlugin.js
https://github.com/webpack/webpack/blob/master/lib/webpack.js#L34